pkg/nb/types.go

```diff
@@ -427,6 +427,12 @@ type GenerateAccountKeysParams struct {
 	Email string `json:"email"`
 }
 
+// UpdateAccountKeysParams is the params of account_api.update_account_keys()
+type UpdateAccountKeysParams struct {
+	Email      string       `json:"email"`
+	AccessKeys S3AccessKeys `json:"access_keys"`
+}
```

Comment:
Do you see the access key and secret key printings in the operator logs after the RPC call?
Please notice PR #1353, in case you see this I think you should wait for the PR to be merged and use the same solution.
cc: @liranmauda

Reply:
I cannot see any of them(access_key and secret_key) printing in operator logs and in core logs printed as some sensitive string. I think we need not to make changes in code for this. Thanks

Reply:
As discussed they currently appear as printing of the RPC request details in the CLI.

Reply:
Great Catch,
It is unrelated to this PR.
@shirady Could you open an Issue for it? 
@achouhan09 We can proceed with this PR

Reply:
@liranmauda sure
@achouhan09 resolving according to the comment above.

Reply:
https://github.com/noobaa/noobaa-operator/pull/1357 should address this issue

Reply:
Thanks Liran, Should I need to update the type to `MaskedString` for accessKeys?

Reply:
After PR #1357 will be merged it will work, you will not need to change something in your code.